lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_computation_client.cpp

```diff
@@ -42,12 +41,11 @@ std::vector<ComputationClient::ComputationPtr> TSComputationClient::Compile(
 std::vector<ComputationClient::DataPtr> TSComputationClient::ExecuteComputation(
     const Computation& computation, lazy_tensors::Span<const DataPtr> arguments,
     const std::string& device, const ExecuteComputationOptions& options) {
-  auto graph =
+  torch::jit::GraphExecutor& graph_executor =
```

Comment:
Don't like `auto&`?

Reply:
Kind of, I only prefer auto when it is a container iterator.

Reply:
Not a big deal. It's up to you. I always use auto for complicated type like `torch::jit::GraphExecutor` above just to save time typing.